nemo/collections/asr/models/ctc_models.py

```diff
@@ -177,17 +188,25 @@ def transcribe(
                             lg = logits[idx][: logits_len[idx]]
                             hypotheses.append(lg.cpu().numpy())
                     else:
-                        current_hypotheses = self._wer.ctc_decoder_predictions_tensor(
-                            greedy_predictions, predictions_len=logits_len, return_hypotheses=return_hypotheses,
+                        current_hypotheses, all_hyp = self.decoding.ctc_decoder_predictions_tensor(
+                            logits, decoder_lengths=logits_len, return_hypotheses=return_hypotheses,
                         )
 
                         if return_hypotheses:
                             # dump log probs per file
                             for idx in range(logits.shape[0]):
                                 current_hypotheses[idx].y_sequence = logits[idx][: logits_len[idx]]
+                                if current_hypotheses[idx].alignments is None:
+                                    current_hypotheses[idx].alignments = current_hypotheses[idx].y_sequence
 
                         hypotheses += current_hypotheses
 
+                        # Keep following for beam search integration
```

Comment:
Are we going to keep this commented section?

Reply:
Yes, for eventual beam search integration. Though its gonna be a breaking change..